vm/chef/cookbooks/phpmyadmin/files/phpmyadmin-setup

```diff
@@ -16,36 +16,30 @@
 
 source /opt/c2d/c2d-utils || exit 1
 
-function apt_retry () {
-  local cmds="apt-get $@"
-  local backoff=10
-  echo "Running cmd: $cmds"
-  while ! ${cmds} 2>&1; do
-    if (( "${backoff}" < 100)); then
-      echo "Error: command failed: ${cmds}"
-      echo "Backoff and retry in $backoff seconds."
-      sleep ${backoff}
-      backoff=$(( backoff * 2 ))
-    else
-      # Fail installation.
-      # Return > 0, so that Launcher communicates the failure to customers.
-      echo "Command failed. Terminating installation."
-      exit 1
-    fi
-  done
-}
-
 readonly installphpMyAdmin="$(get_attribute_value "installphpmyadmin" \
   | tr '[:upper:]' '[:lower:]')"
-readonly mysql_root_password="$(get_attribute_value "mysql-root-password")"
 
 if [[ "${installphpMyAdmin:-}" == "true" ]]; then
   echo "phpMyAdmin has been selected to be installed."
-  echo phpmyadmin phpmyadmin/dbconfig-install boolean true | debconf-set-selections
-  echo phpmyadmin phpmyadmin/mysql/admin-pass password "${mysql_root_password}" | debconf-set-selections
-  echo phpmyadmin phpmyadmin/reconfigure-webserver multiselect apache2 | debconf-set-selections
-  export DEBIAN_FRONTEND='noninteractive'
-  apt_retry -o Dir::Cache::archives="/opt/c2d/downloads/phpmyadmin" install phpmyadmin -yqq
+  setup_folder="/opt/c2d/downloads/phpmyadmin"
+
+  # Set up blowfish_secret parameter
+  secret="$(openssl rand -base64 64 | fold -w 64 | head -n1 | tr -d '\r\n' | tr -dc '[:alnum:]\n\r' | tr '[:upper:]' '[:lower:]')"
```

Comment:
I see a couple comments on line suggesting the secret be 32 characters and that phpMyAdmin can break with a longer string.

https://stackoverflow.com/questions/34961109/what-is-a-blowfish-secret-in-phpmyadmin#comment93928357_34963148

 Is this a problem for us?